### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by warlockrichard**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@
 [![License](http://img.shields.io/:license-Apache%202-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0.txt)
 [![Coverage Status](https://coveralls.io/repos/github/o360/backend/badge.svg?branch=master)](https://coveralls.io/github/o360/backend?branch=master)
 
+Backend project for Open360. The Open360 is a system to create, manage and run surveys in a convenient way for both employees and employers.
```
Comment (maximskripnik) [inline README.md]:
```suggestion
Backend project for Open360. Open360 is a system to create, manage and run surveys in a convenient way for both employees and employers.
```

**Review by maximskripnik — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (warlockrichard) [reply README.md]: Nice catch!

Comment (warlockrichard) [reply README.md]: @elenaustyugova we need to mirror the change on frontend.

**Review by elenaustyugova — APPROVED** [review]

**Review by maximskripnik — APPROVED** [review]
LGTM

**Pull request merged**